docs/core-concepts/authenticating-with-api-keys.mdx

```diff
@@ -0,0 +1,50 @@
+---
+sidebar_label: 'Authenticating with API Keys'
+---
+
+# Authenticating with API Keys
+
+The Regression Games platform provides the ability to generate API keys, which can be used
+to authenticate against our API. This is the recommended way to authenticate against our API
+when connecting external systems to Regression Games.
+
+You can find the **API Keys** manager at the bottom of your [Regression Games account settings](https://play.regression.gg/account).
+
+## Generating an API Key
+
+Enter a unique name for your key and then click **Generate Key**.
+API keys have a default expiration date of 1 year.
```

Comment:
Can we change the default? If not we can just say 'API keys have an expiration date of 1 year.'

Reply:
We cannot change the default from the client today. Removed the "default" bit